src/api/MainApi.ts

```diff
@@ -250,3 +287,26 @@ export default class MainApi {
         return response.data.tags
     }
 }
+
+/**
+ * takes as input a number, let's say "n".
+ * Returns the ISO date string of "n" days ago
+ *
+ * the name of this function can improve D:
+ */
+export function getIsoStringFromDaysPassed(n: number): string {
+    //todays'date
+    const today = new Date()
+    //today's date in milliseconds from the beginning of time
+    const time = today.getTime()
+
+    //milliseconds in n days
+    const timeToSubtract = n * 24 * 60 * 60 * 1000
+
+    //n-days-ago's date in milliseconds from the beginning of time
+    const newTime = time - timeToSubtract
+    //n-days-ago's date
+    const newDate = new Date(newTime)
+
+    return newDate.toISOString()
+}
```

Comment:
forse questa ha senso spostarla in qualche sorta di file nella cartella utils?
con altre eventuali funzioni di util per le date idk ahah

Reply:
Okay